.travis.yml

```diff
@@ -3,7 +3,9 @@ dist: trusty
 
 # we don't need sudo, so can run in a container, which makes startup much
 # quicker.
-sudo: false
+#
+# unfortunately we do require sudo as per https://github.com/travis-ci/travis-ci/issues/8836
```

Comment:
temporarily, hopefully?

Reply:
At some point they should fix the issue, sure.